pledge/index.html

```diff
@@ -127,6 +127,34 @@ <h1 class="ui inverted header">
     </div> <!-- End of card -->
 
 
+<div class="ui card">
+  <div class="content">
+    <img class="right floated mini ui image" 
+            src="https://avatars0.githubusercontent.com/u/37602">
+    <div class="header">
+      Nicholas H.Tollervey
+    </div>
+    <div class="description">
+      Computing, music, philosophy, teaching & writing. Just like this bio: 
```

Comment:
I have now updated you with the advanced HTML codes technology!

On Sat, 7 Nov 2015 at 16:18 Nicholas Tollervey <notifications@github.com>
wrote:

> In pledge/index.html
> <https://github.com/pullrequestday/pullrequestday/pull/1#discussion_r44213898>
> :
>
> > @@ -127,6 +127,34 @@ <h1 class="ui inverted header">
> >      </div> <!-- End of card -->
> >
> >
> > +<div class="ui card">
> > +  <div class="content">
> > +    <img class="right floated mini ui image"
> > +            src="https://avatars0.githubusercontent.com/u/37602">
> > +    <div class="header">
> > +      Nicholas H.Tollervey
> > +    </div>
> > +    <div class="description">
> > +      Computing, music, philosophy, teaching & writing. Just like this bio:
>
> Oops... should be & - I'm doing this in a browser. Not sure how to update
> it. ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pullrequestday/pullrequestday/pull/1/files#r44213898>.
>